var.js

```diff
@@ -0,0 +1,94 @@
+const dbnameDict = {};
```

Comment:
It looks like there is a single json dictionary being returned in this file. Perhaps it would be more concise to be reformatted like following.

```javascript
module.exports = {
  reaction: {
    url_map: {
      ...
    },
    dbname_map: {
      ...
    },
  } ,
  ...
};
```

I think the filename could also be named something more descriptive than `var.js`

Reply:
I addressed this comment in the commit #5534c91